e2e/tests/room/01-createroom.spec.ts

```diff
@@ -261,6 +276,61 @@ describe('Create room screen', () => {
 					.withTimeout(60000);
 				await expect(element(by.id(`rooms-list-view-item-${room}`))).toExist();
 			});
+
+			it('should create a room with non-latin alphabet and do a case insensitive search for it', async () => {
+				const randomValue = random();
+				const roomName = `ПРОВЕРКА${randomValue}`;
+				const roomNameLower = `проверка${randomValue}`;
+
+				await waitFor(element(by.id('rooms-list-view')))
+					.toExist()
+					.withTimeout(10000);
+				await element(by.id('rooms-list-view-create-channel')).tap();
+				await waitFor(element(by.id('new-message-view')))
+					.toBeVisible()
+					.withTimeout(5000);
+				await waitFor(element(by.id('new-message-view-create-channel')))
+					.toBeVisible()
+					.withTimeout(2000);
+				await element(by.id('new-message-view-create-channel')).tap();
+				await waitFor(element(by.id('select-users-view')))
+					.toExist()
+					.withTimeout(5000);
+				await element(by.id('selected-users-view-submit')).tap();
+				await waitFor(element(by.id('create-channel-view')))
+					.toExist()
+					.withTimeout(10000);
+				await waitFor(element(by.id('create-channel-name')))
+					.toBeVisible()
+					.withTimeout(2000);
+				await element(by.id('create-channel-name')).replaceText(roomName);
+				await element(by.id('create-channel-name')).tapReturnKey();
+				await waitFor(element(by.id('create-channel-submit')))
+					.toExist()
+					.withTimeout(2000);
+				await element(by.id('create-channel-submit')).tap();
+				await waitFor(element(by.id('room-view')))
+					.toExist()
+					.withTimeout(60000);
+				await expect(element(by.id('room-view'))).toExist();
+				await waitFor(element(by.id(`room-view-title-${roomName}`)))
+					.toExist()
+					.withTimeout(60000);
+				await expect(element(by.id(`room-view-title-${roomName}`))).toExist();
+				await tapBack();
+				await waitFor(element(by.id('rooms-list-view')))
+					.toExist()
+					.withTimeout(2000);
+				await waitFor(element(by.id(`rooms-list-view-item-${roomName}`)))
+					.toExist()
+					.withTimeout(60000);
+				await expect(element(by.id(`rooms-list-view-item-${roomName}`))).toExist();
+				await searchRoomNonLatin(`${roomNameLower}`, `rooms-list-view-item-${roomName}`);
```

Comment:
Why do you need to specify a lowercase text?
Isn't dealing with case insensitive the whole goal of this PR?

Reply:
Yeap, the room name is uppercase and we are searching only in lowercase. Also, the testID uses the room name in uppercase. So, the test is: we are searching for a room expecting the result is case insensitive.